src/lib.rs

```diff
@@ -28,7 +28,7 @@ pub use zenoh::config as zenoh_config;
 use zenoh::internal::runtime::Runtime as ZRuntime;
 use zenoh::{bytes::ZBytes, pubsub::Subscriber, qos::Priority, Session};
 
-const UATTRIBUTE_VERSION: u8 = 1;
+const UPROTOCOL_MAJOR_VERSION: u8 = 1;
```

Comment:
Why do we rename it to `UPROTOCOL_MAJOR_VERSION`?

Reply:
I'm not sure. Although it is unlikely to happen, do we need to bump the uProtocol version when we have a different way to transform UAttributes into Zenoh attachments?

Reply:
IMHO we do, in order to keep things compatible. Currently, the Zenoh transport spec is part of the uProtocol spec. Thus, the expectation is that if a sender which uses the Zenoh transport that is based on uProtocol 1.x (or 1.6 for that matter), then a receiver using a Zenoh transport based on uProtocol 1.x can correctly receive UMessages.
If we change the Zenoh transport spec, e.g. changing the encoding of the UAttributes, in an incompatible way, then the UProtocol spec version needs to be bumped to a new major version, because that expectation would no longer be met ...